engine/api/mailer/Mailer.php

```diff
@@ -124,6 +124,28 @@ static function createEventPublicationEmail(Event $event)
         Le Nose<br>
         <a href = 'www.nose42.fr' >www.nose42.fr</a>";
         $mailer = Mailer::create();
-        return $mailer->createEmail(Mailer::getGlobalAddress(), $subject, $content);
+        if (!$event->groups->isEmpty()) {
+            $users = [];
+            foreach ($event->groups as $group) {
+                foreach ($group->members as $user) {
+                    if ($user->real_email) {
```

Comment:
I think this will trigger a lot of cascading queries because of lazy loading.

This is often referred to as the n+1 query problem (it basically means a query within a loop)
As the number of user grows, this query will be less and less efficient.
I suggest optimizing this now by creating a custom DQL query.

Digging into the problem a bit further, I think we don't need the name for BCC, only the email. It makes the query a lot simpler to start with, then we can go more complex if we see that we need the name

Reply:
Ok I try to propose a fix